app/models/setting/content.rb

```diff
@@ -145,7 +145,9 @@ def self.default_settings
                   "as long as the registering DMI UUID is not used by another host."),
                true, N_('Host Profile Assume')),
       self.set('host_tasks_workers_pool_size', N_("Amount of workers in the pool to handle the execution of host-related tasks. When set to 0, the default queue will be used instead. Restart of the dynflowd/foreman-tasks service is required."),
-               5, N_('Host Tasks Workers Pool Size'))
+               5, N_('Host Tasks Workers Pool Size')),
+      self.set('applicability_batch_size', N_("Number of host applicability calculations to process per task."),
+               50, N_('Applicability Batch Size'))
```

Comment:
i suspect we'll want something larger asa a default than 50, but let me do some testing!

Reply:
actually after testing, when doing ~1000 profiles, it really didn't make any difference between 30, 50, or 100.  So i think a default of 50 is fine